### ISSUE TITLE
Looks like database is not read/updated (2.5.6, FF 61.0)

### ISSUE THREAD

**Issue by MD-Geist** [brief-rss/brief#376]
This may be related to 368, 375, 374: my feeds do not update and I cannot perform any other operation (add, delete, move,..) on them. I tired the debugging tips from the other bugs, but nothing shows ups. Actually there are only five messages from brief in the console:

start-up:
Brief: opening database in default storage database.js:135:9
Brief: opened database in default storage database.js:168:9
Brief: 187 feeds in database database.js:306:9
Brief: opened database with 1172 entries database.js:89:9

some menu buttons:
Brief: update pref pagePersist to {"startView":"all-items-folder","closedFolders":"_40_160_312_407_","tagList":{"width":""},"sidebar":{"width":"333px","hidden":false},"view":{"mode":"full","filter":"unread"}} prefs.js:34:9 

I also tried moving the ida folder, but that only gave me a db not found message.

What did have some effect, was reinstalling brief. That updated most feeds once. No idea where it picked its feed list from.

Just for good measure, here is the (German) error if I try to add a feed:
Die Dateien unter chrome://brief/content/brief.xhtml?subscribe=https://github.com/brief-rss/brief/commits/master.atom konnten nicht gefunden werden.
Basically a file not found in brief...

The Problem may have started earlier this week, but I have been away till Friday.

More background:
Since the Firefox plugin change, I had to enable storage for brief on each start-up and then reload the plugin. My understanding is, this is actually a Firefox problem, but I wonder, if the unload code may broke my DB.

**Comment by MD-Geist** [comment]
I played a bit with the Firefox console. looks like the DB is actually fine. Unfortunately I do not know enough about JS or FF plugins to continue from here. There should be a way to step through the UI calls with the debugger and see what the back-end is doing? I tried calling Feedview.refresh from the console, but it only printed the source code.

**Comment by ruipin** [comment]
Same is happening to me, haven't been able to figure out what is the cause either...
